client/my-sites/plugins/marketplace/marketplace-test/admin-menu-fetch.tsx

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect } from 'react';
+import { Button, Card } from '@automattic/components';
+import { useSelector, useDispatch } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import useSiteMenuItems from 'calypso/my-sites/sidebar-unified/use-site-menu-items';
+import { getIsRequestingAdminMenu } from 'calypso/state/admin-menu/selectors';
+import { requestAdminMenu } from 'calypso/state/admin-menu/actions';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import { level1ObjectMap } from 'calypso/my-sites/plugins/marketplace/marketplace-test';
+import CardHeading from 'calypso/components/card-heading';
+
+export default function AdminMenuFetch(): JSX.Element {
+	const menuItems = useSiteMenuItems();
+	const isRequestingMenu = useSelector( getIsRequestingAdminMenu );
+	const selectedSiteId = useSelector( getSelectedSiteId );
+	const dispatch = useDispatch();
+
+	const { url: postsPageUrl } =
+		menuItems.find( ( { slug }: { slug: string } ) => slug === 'edit-php' ) ?? {};
+
+	const { url: yoastSeoPageUrl } =
+		menuItems.find( ( { slug }: { slug: string } ) => slug === 'wpseo_dashboard' ) ?? {};
+
+	useEffect( () => {
+		if ( ! postsPageUrl || ! yoastSeoPageUrl ) {
+			selectedSiteId && dispatch( requestAdminMenu( selectedSiteId ) );
```

Comment:
Should `selectedSiteId` just be inside the `if` logic above?

Reply:
Sorry we do not need to review these pages. They are meant as an easy debug panel for developers.

Reply:
Let's try to make this more explicit. Also, I don't quite like this - any code that goes into Calypso should be reviewed and approved. Otherwise, it's very easy for someone who's new to Calypso to borrow from this "non-production" code in their project and end up with buggy or suboptimal code in production 😉